presto-docs/src/main/sphinx/release/release-346.md

```diff
@@ -0,0 +1,91 @@
+# Release 345 (XX Nov 2020)
+
+## General Changes
+
+* Add support for `RANGE BETWEEN <value> PRECEDING AND <value> FOLLOWING` window frames. ({issue}`609`)
+* Add support for window frames based on `GROUPS`. ({issue}`5713`)
+* Add support for {func}`extract` with `TIMEZONE_HOUR` and `TIMEZONE_MINUTE` for `time with time zone` values. ({issue}`5668`)
+* Add SQL syntax for `GRANT` and `REVOKE` on schema. This is not yet used by any connector. ({issue}`4396`)
+* Add `ALTER TABLE SET AUTHORIZATION` syntax to allow to change the table owner. ({issue}`5717`)
+* Make `EXPLAIN` more readable for queries containing `timestamp` or `timestamp with time zone`  constants. ({issue}`5683`)
+* Improve performance for queries with inequality conditions. ({issue}`2674`)
+* Improve performance of queries with uncorrelated ``IN`` clauses. ({issue}`5582`)
+* Do not write min/max statistics for `timestamp` columns. ({issue}`5858`)
+* Fix incorrect handling of negative offsets for the `time with time zone` type. ({issue}`5696`)
+* Fix incorrect result when casting `time(p)` to `timestamp(p)` for precisions higher than 6. ({issue}`5736`)
+* Fix incorrect query results when comparing a `timestamp` column with a `timestamp with time zone` constant. ({issue}`5685`)
+* Fix improper table alias visibility for queries that select all fields. ({issue}`5660`)
+* Fix failure when query parameter appears in a lambda expression. ({issue}`5640`)
+* Fix failure for queries containing `DISTINCT *` and fully-qualified column names in the `ORDER BY` clause. ({issue}`5647`)
+* Fix planning failure for certain queries involving `INNER JOIN`, `GROUP BY` and correlated subqueries. ({issue}`5846`)
+* Fix issue when `EventListener` was sometimes not called if query was aborted in initial phase of execution. ({issue}`5815`)
```

Comment:
@losipiuk, what's the user-visible effect? Also, this description is too low-level. Maybe something like "Fix recording of query completion event when query is aborted early".

Reply:
I think your suggestion is nice. This is user visible as events are consumed by users code (via provided `EventListener` implementations)